starter-code/pizza.js

```diff
@@ -1 +1,41 @@
 // Write your Pizza Builder JavaScript in this file.
+
+
+//ONLOAD
+$(".pep").toggle();
```

Comment:
You have a function for do this:
```
pepperoni()
```
Call it instead of writing the same code again. The same for the rest of starting configuration.